csm/core/blogic/const.py

```diff
@@ -914,4 +914,5 @@
 AGENT_ENDPOINTS = 'CSM_SERVICE>CSM_AGENT>endpoints'
 AGENT_HOST = 'CSM_SERVICE>CSM_AGENT>host'
 AGENT_PORT = 'CSM_SERVICE>CSM_AGENT>port'
-AGENT_BASE_URL = 'CSM_SERVICE>CSM_AGENT>base_url'
\ No newline at end of file
+AGENT_BASE_URL = 'CSM_SERVICE>CSM_AGENT>base_url'
+TELEMENTRY_AUTH = 'STATS>telemetryAuth'
```

Comment:
If you are not using TELEMENTRY_AUTH.. then please remove it.

Reply:
removed